core/lexicon/en/policy.inc.php

```diff
@@ -55,10 +56,11 @@
 $_lang['policy_remove_multiple_confirm'] = 'Are you sure you want to remove these Access Policies? This is irreversible.';
 $_lang['policy_update'] = 'Update Policy';
 $_lang['policy_template'] = 'Policy Template';
-$_lang['policy_template.desc'] = 'A Policy Template defines which Permissions will show up in the Permissions grid when editing a specific Policy. You can add or remove specific Permissions from this template below. Note that removing a Permission from a Template will remove it from any Policies that use this Template.';
+$_lang['policy_template_desc'] = 'A Policy Template defines which Permissions will show up in the Permissions grid when editing a specific Policy. You can add or remove specific Permissions from this template below. Note that removing a Permission from a Template will remove it from any Policies that use this Template.';
```

Comment:
If you're wondering (just like me 😅) what the change is: The `.` is replaced in the lexicon key!

Reply:
It is done just to simplify logic in updater scripts. And a side effect, make lexicon keys more consistent. :)

Reply:
It's a good change, I just had a hard time spotting it haha.